example/Supfile

```diff
@@ -29,6 +29,8 @@ networks: # Groups of hosts
       - ubuntu@stg.example.com
 
   prod:
+    inventory:
+      - for i in 4 5 6; do echo "$USER@prod$i.example.com\n"; done
```

Comment:
@stengaard I wonder as well like @VojtechVitek, why did you choose to make it an array of commands? I believe this should just take a single command.